3761/1/table.rb

```diff
@@ -0,0 +1,32 @@
+require 'terminal-table'
+# create and display table with rating of gems
+class Table
+  def initialize(array_of_hash, top)
+    @array_of_hash = array_of_hash
+    @top = top
+    @table = create
+  end
+
+  def write
+    puts @table
+  end
+
+  private
+
+  def create
+    table = Terminal::Table.new
+    0.upto(@top - 1) do |iter|
+      table.add_row gener_row_info(@array_of_hash[iter])
+    end
+    table.style = { border_top: false, border_bottom: false }
+    table
+  end
+
+  def gener_row_info(data)
```

Comment:
You can generate this data for each class you call this table. But better to create some Presenter or Serializer for Gem